### PR TITLE
Flock data

### PR DESCRIPTION
**Pull request by AbdAllahAboElMakarem**

### CODE REVIEW

File: mobileapp/lib/App/register.dart [hunk 1]
```diff
@@ -53,14 +55,12 @@ class _RegisterPage extends State<RegisterPage> {
   }
 
   String validateName(String value) {
-    if (value == null || value.trim().isEmpty)
-      return "Name is required!";
+    if (value == null || value.trim().isEmpty) return "Name is required!";
     return null;
```
Comment (mabdelaal86) [inline mobileapp/lib/App/register.dart:59]: consider indentation here

File: mobileapp/lib/App/register.dart [hunk 2]
```diff
@@ -53,14 +55,12 @@ class _RegisterPage extends State<RegisterPage> {
...
   }
 
   String validatePhone(String value) {
-    if (value == null || value.trim().isEmpty)
-      return "Phone is required!";
+    if (value == null || value.trim().isEmpty) return "Phone is required!";
     return null;
```
Comment (mabdelaal86) [inline mobileapp/lib/App/register.dart:64]: consider indentation here

File: mobileapp/lib/App/settings.dart [hunk 3]
```diff
@@ -24,16 +26,16 @@ class _SettingsPage extends State<SettingsPage> {
           builder: (context, snapshot) {
             if (snapshot.hasError)
               return buildMessage(snapshot.error.toString());
-            if (!snapshot.hasData)
-              return buildMessage("Loading...");
+            if (!snapshot.hasData) return buildMessage("Loading...");
```
Comment (mabdelaal86) [inline mobileapp/lib/App/settings.dart:29]: At least make all statements follow the same indentation way

**Review by mabdelaal86 — COMMENTED** [review]
Nice work. Justy need to make sure it works on emulator

**Comment by AbdAllahAboElMakarem** [comment]
> Nice work. Justy need to make sure it works on emulator
thanks for reading the code and commenting on it
I am having some issues with the emulator on my device and switched on, I am working on the emulator online, but it is not working, or I have come up with a solution so far
the code works
the spaces in the code are when you press the button (ctrl + s). The code is saved in the standard way in the framework.

**Review by mabdelaal86 — APPROVED** [review]
Done

**Pull request merged**